Libraries/Image/RCTLocalAssetImageLoader.m

```diff
@@ -50,7 +51,49 @@ - (RCTImageLoaderCancellationBlock)loadImageForURL:(NSURL *)imageURL
     }
 
     NSString *imageName = RCTBundlePathForURL(imageURL);
-    UIImage *image = [UIImage imageNamed:imageName];
+    NSBundle *targetBundle = [NSBundle mainBundle];
+    
+    if (bundlePath) {
+      NSString *targetBundleName = RCTBundlePathForURL([NSURL URLWithString:bundlePath]);
+      NSString *targetBundleNameWithExtension = [NSString stringWithFormat:@"%@%@", targetBundleName, @".bundle"];
+      NSMutableArray *bundles = [[NSBundle allBundles] mutableCopy];
+      
+      // Get any bundle files included in the mainBundle and add them to the array
+      for (NSString *path in [[NSBundle mainBundle] pathsForResourcesOfType:@"bundle" inDirectory:nil]) {
```

Comment:
No, they can't change at runtime so this should be cached; I'll see what I can plug in quickly.